babel.config.js

```diff
@@ -15,4 +15,9 @@ module.exports = {
     '@babel/plugin-transform-flow-strip-types',
     '@babel/plugin-transform-react-jsx',
   ],
+  env: {
+    development: {
+      ignore: [/__tests__/],
```

Comment:
Do you import something from `__tests__`? Babel shouldn't run otherwise I guess

Reply:
I don't want to transpile files in `__tests__` when building the packages, but I do want files in `__tests__` to be transpiled when testing. I don't want to specify `--ignore` flag when running babel, due to issues on windows.